content/articles/account-subscription-balance.md

```diff
@@ -6,39 +6,43 @@ categories:
 ---
 
 # Account Balance
+The Account page, linked in the top right corner of the Domain List, provides comprehensive information about the account's billing and plan status. 
 
-The account page has a full section dedicated to relevant information about your billing status. This information includes:
+## Checking your account billing details
+1. Choose the relevant account in the top bar on your dashboard, then click **Account** at the top right.
+1. On the account page, you can find your account balance under the **Billings and plans** tab on the left side of the screen.
 
-- **Next payment**
-  When the next subscription fee will be charged to the credit card on file.
+## Billing details breakdown
+On the **Billings and plans** page, you can view: 
 
-- **Amount due**
-  The total amount of the subscription renewal.
+### Estimated amount due
+The total estimated amount due and the date it will be charged to the card on file. Next to the balance, you'll see the date your next payment is due.
 
-  It's broken up into three different components that you may be charged for: _plan price_, active zones_ and _email forwards_.
+This amount is broken up into:
+- [DNS queries](/articles/dns-query-limits/) 
+- Subscription cost
+- Active zones
+- Extra [seats](/articles/managing-seats/) (on the Teams plan and higher)
+- [Email forwards](/articles/email-forwarding/)
+- Account credits (when applicable)
 
-- **Subscription balance**
-  The current balance of your account. The amount can be:
+**SCREENSHOT**
```

Comment:
https://drive.google.com/drive/u/3/search?q=type:image%20parent:1Y9yg2ZWQDXY8cjvNf6wUiZT8SnPTJ9EP

Reply:
have added the screenshot and pushed up